playbooks/gluster-cluster/README

```diff
@@ -0,0 +1,13 @@
+This example shows how to deploy a Gluster cluster end-to-end.
+
+The cluster-vars.yml file contains variables for setting the backend disks using
+LVM thinpool and creating a Gluster volume. The playbook illustrates a 2x2
+replicate volume, but any volume type can be created.
+Please refer: https://github.com/gluster/gluster-ansible-cluster
+
+Usage:
+To run the playbook create an inventory file containing the hosts in question.
+Edit the inventory file to suit your needs. And run the playbook:
+
+$ ansible-playbook -i path-to-inventory gluster_vol_2x2.yml
```

Comment:
rename gluster_vol_2x2.yml to gluster_vol_create.yml?

Reply:
Ack.